modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -300,6 +302,7 @@ object Build {
             buildClient,
             actualCompiler,
             buildTests,
+            suppressDirectivesInMultipleFilesWarning,
```

Comment:
I'm thinking that instead of passing it separately in each of the sub-commands, you could add it as a field in `scala.build.options.BuildOptions` and pass it in `SharedOptions.buildOptions(...)`